collectors/bzimport/convertors.py

```diff
@@ -595,6 +595,34 @@ def major_incident_state(self):
 
         return valid_pairs.get(flags_from_bz, Flaw.FlawMajorIncident.INVALID)
 
+    @cached_property
+    def requires_summary(self):
+        """
+        A requires_summary state created from the requires_doc_text flag.
+        """
+        # Default values
+        status = ""
+        setter = ""
+
+        # Sets values from Bugzilla
+        for flag in self.flags:
+            if flag["name"] == "requires_doc_text":
+                setter = flag["setter"]
+                status = flag["status"]
+
+        # this combination is set automatically by BZ when Doc Text is added
+        if setter == "bugzilla@redhat.com" and status == "+":
+            return Flaw.FlawRequiresSummary.REQUESTED
+
+        pairs = {
+            "": Flaw.FlawRequiresSummary.NOVALUE,
+            "-": Flaw.FlawRequiresSummary.REJECTED,
+            "?": Flaw.FlawRequiresSummary.REQUESTED,
+            "+": Flaw.FlawRequiresSummary.APPROVED,
+        }
+
+        return pairs.get(status)
```

Comment:
I suggest using `pairs[status]`. That way, it crashes outright if there's an unexpected symbol. Shouldn't happen though, because BZ docs specify only ?+- in https://www.bugzilla.org/docs/2.18/html/flags-overview.html